pyproject.toml

```diff
@@ -205,6 +204,10 @@ warn_return_any = true
 warn_unreachable = true
 warn_unused_ignores = true
 
+[[tool.mypy.overrides]]
```

Comment:
Any chance we could scope the override to docs only?

Reply:
I can't find the option to do both override of missing import of specific module and only on specific folder/pattern.

Reply:
No big deal then :slightly_smiling_face: